src/api/liskApi.js

```diff
@@ -51,175 +50,174 @@ const livePort = 8000;
- * @param node string
- * @return {object}
- */
+		return NetHash;
+	}
 
-LiskAPI.prototype.setNode = function setNode(node) {
-	this.currentPeer = node || privateApi.selectNode.call(this);
-	return this.currentPeer;
-};
+	/**
+	 * @method getPeers
+	 * @return {object}
+	 */
+
+	getPeers() {
+		return {
+			official: this.defaultPeers.map(node => ({ node })),
+			ssl: this.defaultSSLPeers.map(node => ({ node, ssl: true })),
+			testnet: this.defaultTestnetPeers.map(node => ({ node, testnet: true })),
+		};
+	}
 
-/**
- * @method setTestnet
- * @param testnet boolean
- */
+	/**
+	 * @method setNode
+	 * @param node string
+	 * @return {object}
+	 */
 
-LiskAPI.prototype.setTestnet = function setTestnet(testnet) {
-	if (this.testnet !== testnet) {
-		this.bannedPeers = [];
+	setNode(node) {
+		this.currentPeer = node || privateApi.selectNode.call(this);
+		return this.currentPeer;
 	}
-	this.testnet = testnet;
-	this.port = testnet ? testPort : livePort;
 
-	privateApi.selectNode.call(this);
-};
+	/**
+	 * @method setTestnet
+	 * @param testnet boolean
+	 */
 
-/**
- * @method setSSL
- * @param ssl boolean
- */
+	setTestnet(testnet) {
+		if (this.testnet !== testnet) {
+			this.bannedPeers = [];
+		}
+		this.testnet = testnet;
+		this.port = testnet ? testPort : livePort;
 
-LiskAPI.prototype.setSSL = function setSSL(ssl) {
-	if (this.ssl !== ssl) {
-		this.ssl = ssl;
-		this.bannedPeers = [];
 		privateApi.selectNode.call(this);
 	}
-};
-
-/**
- * @method getAddressFromSecret
- * @param secret
- *
- * @return keys object
- */
-
-LiskAPI.prototype.getAddressFromSecret = function getAddressFromSecret(secret) {
-	const accountKeys = cryptoModule.getKeys(secret);
-	const accountAddress = cryptoModule.getAddress(accountKeys.publicKey);
-
-	return {
-		address: accountAddress,
-		publicKey: accountKeys.publicKey,
-	};
-};
 
-/**
- * @method broadcastSignedTransaction
- * @param transaction
- * @param callback
- *
- * @return API object
- */
-
-LiskAPI.prototype.broadcastSignedTransaction = function broadcastSignedTransaction(
-	transaction, callback,
-) {
-	const request = {
-		requestUrl: `${privateApi.getFullURL.call(this)}/api/transactions`,
-		nethash: this.nethash,
-		requestParams: { transaction },
-	};
+	/**
+	 * @method setSSL
+	 * @param ssl boolean
+	 */
+
+	setSSL(ssl) {
+		if (this.ssl !== ssl) {
+			this.ssl = ssl;
+			this.bannedPeers = [];
+			privateApi.selectNode.call(this);
+		}
+	}
 
-	privateApi.sendRequestPromise.call(this, POST, request).then(result => callback(result.body));
-};
+	/**
+	 * @method broadcastSignedTransaction
+	 * @param transaction
+	 * @param callback
+	 *
+	 * @return API object
+	 */
+
+	broadcastSignedTransaction(
+		transaction, callback,
+	) {
+		const request = {
+			requestUrl: `${privateApi.getFullURL.call(this)}/api/transactions`,
+			nethash: this.nethash,
+			requestParams: { transaction },
+		};
+
+		privateApi.sendRequestPromise.call(this, POST, request).then(result => callback(result.body));
```

Comment:
Perhaps beyond the scope of this PR but this method should match the `Promise`/callback pattern of `sendRequest`.